lib/core/utils/parse-crossorigin-stylesheets.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Parse cross-origin stylesheets
+ *
+ * @param {String} url url from which to fetch stylesheet
+ * @param {Object} options options object from `axe.utils.parseStylesheets`
+ * @param {Array<Number>} priority sheet priority
+ * @param {Array<String>} importedUrls urls of already imported stylesheets
+ * @param {Boolean} isCrossOrigin boolean denoting if a stylesheet is `cross-origin`
+ * @returns {Promise}
+ */
+axe.utils.parseCrossOriginStylesheets = function parseCrossOriginStylesheets(
+	url,
+	options,
+	priority,
+	importedUrls,
+	isCrossOrigin
+) {
+	const axiosOptions = {
+		method: 'get',
+		url,
+		timeout: options.timeout
+	};
+
+	importedUrls.push(url);
+	return axe.imports.axios(axiosOptions).then(({ data }) => {
```

Comment:
Do we need a `catch`? None of the calls to this function handle it.

Reply:
We fail safely if any preload fails, see - https://github.com/dequelabs/axe-core/pull/1339/files#diff-093a28bf6f7487126983e1ddcb9fc288R445, hence the top-level catch.